PROJECT

```diff
@@ -22,7 +22,7 @@ resources:
   controller: true
   domain: splunk.com
   group: enterprise
-  kind: ClusterMaster
+  kind: IndexerCluster
```

Comment:
Why are these 'Kind's changing? same with other places as well.

Reply:
The git view became a bit confusing here. If we compare the whole file it's easier to understand the changes. In short:
```
*Master -> Only supported in V3
*Manager -> supported in V4
All others(Indexer,SH,etc) -> supported in both V3 and V4
```
This was caused because we had to run `operator-sdk create api ` again so the order of items changed, but git shows the changes a bit convoluted.